src/gc.c

```diff
@@ -1196,6 +1196,18 @@ static NOINLINE jl_taggedvalue_t *add_page(jl_gc_pool_t *p) JL_NOTSAFEPOINT
     return fl;
 }
 
+// called into by LLVM-generated code
+JL_DLLEXPORT jl_value_t *jl_gc_pool_alloc_outer(jl_ptls_t ptls, int pool_offset,
+                                          int osize)
+{
+    jl_value_t *val = jl_gc_pool_alloc(ptls, pool_offset, osize);
+    // zero out the type tag so it's not garbage
```

Comment:
Why is this needed?

Reply:
we took it out, but I think something was lost in a rebase… fix coming shortly

Reply:
Yeah, repushed the commit; sorry about that. It's definitely not needed 👍